styles/styles.css

```
@@ -0,0 +1,328 @@
+@font-face {
+  font-family: 'inter';
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/inter-400.woff2") format("woff2");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'inter';
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/inter-700.woff2") format("woff2");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'inter';
+  font-style: normal;
+  font-weight: 900;
+  src: url("../fonts/inter-900.woff2") format("woff2");
+  font-display: swap;
+}
+
+body {
+  font-family: 'inter', sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #565C66;
+  background-color: #FEAFC3;
+}
+
+.page-header {
+  color: #2D3440;
+}
+
+.navigation-link {
+  font-family: 'inter' sans-serif;
```

Тут должна быть запятая, такое значение не валидно
```suggestion
  font-family: 'inter', sans-serif;
```

---

поправила